code/datums/actions/xeno_action.dm

```diff
@@ -136,9 +136,11 @@
 	return cooldown_timer
 
 
-/datum/action/xeno_action/proc/add_cooldown()
+/datum/action/xeno_action/proc/add_cooldown(cooldown_override = FALSE)
```

Comment:
either that or a number since it's not a TRUE FALSE
```suggestion
/datum/action/xeno_action/proc/add_cooldown(cooldown_override)
```

Reply:
Done.